lib/commands/general.js

```diff
@@ -47,6 +47,21 @@ commands.getWindowSize = async function () {
   return await this.uiautomator2.jwproxy.command('/window/current/size', 'GET', {});
 };
 
+commands.execute = async function (script, args) {
```

Comment:
this might override the magic "mobile: shell" endpoint. can we somehow merge the stuff?

Reply:
hmmm, that one was added while working on this code. i'll check it out and make sure they don't clash.